Libraries/Image/RCTImageLoader.m

```diff
@@ -74,8 +74,7 @@ + (void)loadImageWithTag:(NSString *)imageTag callback:(void (^)(NSError *error,
           // doesn't spike the memory up during the process.
           @autoreleasepool {
             ALAssetRepresentation *representation = [asset defaultRepresentation];
-            ALAssetOrientation orientation = [representation orientation];
```

Comment:
is it OK to ignore the orientation info? seems like this metadata should be threaded through to all the way to coregraphics.